### DISCUSSION TITLE
Annotated tags message display is off

### DISCUSSION THREAD

**Discussion by sylver** [community/community#61136] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

Hi,

I'm not sure this is intended (if so why ?) or a bug (if so, where should I report it ?), but annotated tag message display is off on Github.

Taking this simple example message :

```
> git show v1.1.0       
tag v1.1.0
Tagger: John Doe <john.doe@email.org>
Date:   Tue Jul 18 12:33:24 2023 +0200

This is an annotated tag

features:
- foo
- bar

fixes:
- ber
-----BEGIN PGP SIGNATURE-----
[...]
```

Here is how this is display on Github : 

<img src="https://github.com/community/community/assets/3833987/5dd4ce95-1f1c-40ff-9a12-b9d4bf9aa9ec" width="400">

- Empty line separator between title and description is gone
- Second line has been prefixed with 6 (?) spaces
- The rest seems ok

Am I missing something here ?

**Comment by JakubFranek** [comment]
I am seeing this issue as well and it is driving me nuts. I can't find any info online about whether this is intended or not, but I hope it isn't, because I see no reason why the second line has to be always indented.

**Comment by la-cc** [comment]
GitHub may render commits, tags, and other text in its own way, sometimes leading to discrepancies like what you've observed.

Regarding your issue:

Empty line separator: GitHub might be ignoring or not properly rendering empty lines in certain contexts. This might be a design decision, or possibly a bug.
Spaces before the second line: The extra spaces might be due to the way GitHub formats code blocks or text. If your message was indented in your git tool, GitHub might be retaining that formatting.
Here are a couple of things you could try:

Check your text editor's settings, such as line endings (LF vs CRLF) and indentation, to see if there are any configurations that might be affecting the display on GitHub.

If you believe this to be a bug, you can report it to GitHub Support. They have a contact form available at https://support.github.com/contact/

**Comment by sylver** [comment]
@la-cc thanks for having taken the time to think about it.

Thing is, the rendering problem on Github cannot be explained by what you're suggesting, that's raw text, and that's stored in a git annotated tag message, there is no editor involved, that's CI generated text (on a basic linux runner).
As I showed in my first post, the tag message is properly stored in git, so there is no reason github would change the formatting on its own (especially when using a specific font to emphasize it is raw text)

I'll contact the support then.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @sylver ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. I've gone ahead and moved it for you. Good luck!

**Reply by sylver** [reply]
Oops, sorry about that.
Thanks for pointing that out and having moved my thread to the right topic 🙏

**Reply by queenofcorgis** [reply]
@sylver no need to apologize, just wanted to let you know why your post was moved ❤️ Thank you for posting the update from support as well!

**Comment by sylver** [comment]
Issue confirmed by support.
So let's wait for a fix eventually.

Thank you all.

<img src="https://github.com/community/community/assets/3833987/c3099857-6a89-42d9-97bd-95ac53403aa1" width="400">

**Comment by vaindil** [comment, marked as answer]
Hi folks, a fix was pushed for this a little earlier today, tag messages should no longer appear incorrectly formatted. Please let us know if you see the problem again!

**Reply by sylver** [reply]
Whoa that was fast !
Thank you so much :pray: ! (That wasn't the end of the world, but really annoying nonetheless)
